### PR TITLE
Adding isclose as additional checker

### PR DESCRIPTION
**Pull request by loumalouomega**

### CODE REVIEW

**Comment by loumalouomega** [comment]
I added this because some tests (long numbers with almost no decimal part) were not working with the current implementation with AssertAlmostEqual

File: kratos/python_scripts/from_json_check_result_process.py [hunk 1]
```diff
@@ -104,21 +104,42 @@ def ExecuteFinalizeSolutionStep(self):
                     if value_scalar:
                         values_json = self.data["NODE_"+str(node.Id)][out.GetString() ]
                         value_json = linear_interpolation(time, input_time_list, values_json)
-                        self.assertAlmostEqual(value, value_json, msg=("Error checking node "+str(node.Id)+" "+out.GetString()+" results."), delta=tol)
+                        if (self.iscloseavailable == True):
+                            isclosethis = math.isclose(value, value_json, rel_tol=tol, abs_tol=tol)
```
Comment (rubenzorrilla) [inline kratos/python_scripts/from_json_check_result_process.py]: Should the relative and the absolute tolerances have the same value?

File: kratos/python_scripts/from_json_check_result_process.py [hunk 2]
```diff
@@ -104,21 +104,42 @@ def ExecuteFinalizeSolutionStep(self):
...
-                        self.assertAlmostEqual(value[1], value_json, msg=("Error checking node "+str(node.Id)+" "+out.GetString()+" Y-component results."), delta=tol)
-                        # Z-component
-                        values_json = self.data["NODE_"+str(node.Id)][out.GetString()  + "_Z"]
-                        value_json = linear_interpolation(time, input_time_list, values_json)
-                        self.assertAlmostEqual(value[2], value_json, msg=("Error checking node "+str(node.Id)+" "+out.GetString()+" Z-component results."), delta=tol)
+                        if (self.iscloseavailable == True):
+                            # X-component
+                            values_json = self.data["NODE_"+str(node.Id)][out.GetString()  + "_X"]
+                            value_json = linear_interpolation(time, input_time_list, values_json)
+                            isclosethis = math.isclose(value[0], value_json, rel_tol=tol, abs_tol=tol)
```
Comment (rubenzorrilla) [inline kratos/python_scripts/from_json_check_result_process.py]: The same for the vector variable case.

File: kratos/python_scripts/from_json_check_result_process.py (hunk 1 above)
Comment (loumalouomega) [reply kratos/python_scripts/from_json_check_result_process.py]: I was thinking in that, but that means to change values...I have an idea

**Comment by loumalouomega** [comment]
@rubenzorrilla Done

**Review by rubenzorrilla — APPROVED** [review]

**Pull request merged**